config/confd/conf.d/cloudfront-pk.pem.toml

```diff
@@ -0,0 +1,7 @@
+[template]
+src = "cloudfront-pk.pem.tmpl"
+dest = "/etc/docker/cloudfront-pk.pem"
+keys = [
+  "/resin/registry2/s3/cloudfront/privatekey",
```

Comment:
Same here -- I think you need to request `/resin/registry2/s3` for this to not fail with an error.

Reply:
Good point. We definitely want CloudFront config as optional. Addressed :+1: